ui/js/directives/treeherder/resultsets.js

```diff
@@ -39,17 +39,11 @@ treeherder.directive('thAuthor', function () {
     return {
         restrict: "E",
         link: function (scope, element, attrs) {
-            var userTokens = attrs.author.split(/[<>]+/);
-            var email = "";
-            if (userTokens.length > 1) {
-                email = userTokens[1];
-            }
-            scope.authorName = userTokens[0].trim();
-            scope.authorEmail = email;
+            scope.authorEmail = attrs.author.split(/[<>]+/)[0].trim();
```

Comment:
For every repository I've checked (except for the now-defunct "servo" repo[1]), `attrs.author` is *only* the plain email address. In the servo repo, it's in the form "name \<email\>".

Thus, after the split, only the old, defunct servo repository has two pieces. Everything else only has the one for the email address. If you look at pushes on servo, authorEmail is pretty much always going to be "bors-servo", but everywhere else, it would be the actual author email.

So this PR in total changes:
* the tooltip from showing the email address to just a generic "this user"
* the push titlebar from showing author name (in most cases, this is actually the email address) to always showing the email address

1. https://treeherder.mozilla.org/#/jobs?repo=servo